/beamform2:
    ros__parameters:
        auto_connect: false
        output_type: 1
        # 0 BOTH
        # 1 ONLY JACK
        # 2 ONLY ROS
        write_file: true
        write_file_path: ''
        write_xrun: true
        ros_output_sample_rate: 48000
